src/AutoUI/schemaOps.ts

```diff
@@ -138,20 +140,20 @@ export interface ActionData<T> {
 	checkedState?: CheckedState;
 }
 
-export const isJson = (str: string) => {
+export const isJson = (str: string): boolean => {
```

Comment:
The return typing here is needed?

Reply:
not sure why this was added, but removing it does not add it back. removed all